### DISCUSSION TITLE
Disable front led status indicators

### DISCUSSION THREAD

**Discussion by ghost** [seerge/g-helper#1159] [Ideas]
Hi. Is this possible to create a switch to turn on/off indicators about battery/disk status on the top of the laptop (I have them on G15, but I'm sure that they are on other laptops as well)? They are a little bit distracting, especially if you are trying to work in the dark

**Comment by seerge** [comment]
No, they are always ON unfortunately

**Reply by ghost** [reply]
Oh, ok. Thanks for your work
